docs/jwst/firstframe/description.rst

```diff
@@ -0,0 +1,8 @@
+
+Description
+===========
+
+The first frame correction step flags the first group as bad (the GROUP data
```

Comment:
Two suggestions. Mention right off the bat that this is a MIRI-only correction, e.g. "The MIRI first frame correction ...". Second, wouldn't hurt to be explicit and say "first group in every integration".

Reply:
Done.